thinkhazard/static/js/report.js

```diff
@@ -31,6 +31,19 @@
     map.getTargetElement().style.cursor = hit ? 'zoom-in' : '';
   });
 
+  // drill down
+  map.on('click', function(e) {
+    var feature = map.forEachFeatureAtPixel(e.pixel, function(feature) {
+      return feature;
+    });
+    if (feature) {
+      var code = feature.get('code');
+      if (code) {
```

Comment:
I'm not completely sure what the server sends. But you're right, we can assume that this is always set. To be removed.